browser_calls_flask/config.py

```diff
@@ -6,10 +6,12 @@
 
 basedir = os.path.abspath(os.path.dirname(__file__))
 
+sqllite_uri = 'sqlite:///' + os.path.join(basedir, 'prod.sqlite')
```

Comment:
Nit: this doesn't need to be a variable if you're only using it in one location (like the DevelopmentConfig SQLALCHEMY_DATABASE_URI below).